src/lib/crypto/kimchi_bindings/js/native/header-d.ts

```diff
@@ -0,0 +1,9 @@
+// Header section
+// To edit this section, look for 
+// ./src/lib/crypto/kimchi_bindings/js/native/header-d.ts
+// This file gets auto-included in the generated plonk-napi types to supplement
+// external pointer types.
+
+type WasmPastaFpPlonkIndex = {};
```

Comment:
This means every time we have any  `External<T>` we need to include a `type T = {}` here? Just confirming, looks like what you shared with us during this week call.